src/client/tile.h

```diff
@@ -325,6 +328,9 @@ struct TileSpec
 			&& emissive_light == other.emissive_light;
 	}
 
+	//! If true, the tile rotation is ignored.
```

Comment:
for texturing purposes only, right?
(comment should explain this)

Reply:
*tile* rotation, not *node* rotation

Reply:
oh right, make sense